config.yaml

```diff
@@ -106,109 +108,109 @@ drug:
 evidence:
   gs_downloads_latest:
   - bucket: otar000-evidence_input/CancerBiomarkers/json
-    output_filename: cancerbiomarkers-{suffix}.json.gz
-    output_spark_dir: cancerbiomarkers-{suffix}
+    output_filename: cancerbiomarkers.json.gz
+    output_spark_dir: cancerbiomarkers
     path: evidence-files
   - bucket: otar000-evidence_input/Orphanet/json
-    output_filename: orphanet-{suffix}.json.gz
-    output_spark_dir: orphanet-{suffix}
+    output_filename: orphanet.json.gz
+    output_spark_dir: orphanet
     path: evidence-files
   - bucket: otar000-evidence_input/ClinGen/json
-    output_filename: clingen-{suffix}.json.gz
-    output_spark_dir: clingen-{suffix}
+    output_filename: clingen.json.gz
+    output_spark_dir: clingen
     path: evidence-files
   - bucket: otar000-evidence_input/CRISPR/json
-    output_filename: crispr-{suffix}.json.gz
-    output_spark_dir: crispr-{suffix}
+    output_filename: crispr.json.gz
+    output_spark_dir: crispr
     path: evidence-files
   - bucket: otar006-reactome
-    output_filename: reactome-{suffix}.json.gz
-    output_spark_dir: reactome-{suffix}
+    output_filename: reactome.json.gz
+    output_spark_dir: reactome
     path: evidence-files
   - bucket: otar007-cosmic
-    output_filename: cosmic-{suffix}.json.gz
-    output_spark_dir: cosmic-{suffix}
+    output_filename: cosmic.json.gz
+    output_spark_dir: cosmic
     excludes: hallmarks
     path: evidence-files
   - bucket: otar000-evidence_input/ChEMBL/json
-    output_filename: chembl-{suffix}.json.gz
-    output_spark_dir: chembl-{suffix}
+    output_filename: chembl.json.gz
+    output_spark_dir: chembl
     path: evidence-files
   - bucket: otar000-evidence_input/Genetics_portal/json/
-    output_filename: genetics-portal-evidences-{suffix}.json.gz
-    output_spark_dir: genetics-portal-evidences-{suffix}
+    output_filename: genetics-portal-evidences.json.gz
+    output_spark_dir: genetics-portal-evidences
     path: evidence-files
   - bucket: otar010-atlas
-    output_filename: atlas-{suffix}.json.bz2
-    output_spark_dir: atlas-{suffix}
+    output_filename: atlas.json.bz2
+    output_spark_dir: atlas
     path: evidence-files
   - bucket: otar011-uniprot
-    output_filename: uniprot-{suffix}.json.gz
-    output_spark_dir: uniprot-{suffix}
+    output_filename: uniprot.json.gz
+    output_spark_dir: uniprot
     path: evidence-files
   - bucket: otar012-eva
-    output_filename: eva-{suffix}.json.gz
-    output_spark_dir: eva-{suffix}
```

Comment:
I don't think we are ready to make this move, our dataproviders are sending us data following the old pattern with the timestamps. Or you can remove the timestamps one by one. 

If we have to the data team can update our parsers so no data stamp is included in the output files, but that is just a fraction of the evidence input.

Reply:
This is fine, as the data is still collected taking the timestamps into account, but they are not propagated to the output file in the ETL's input folder.

Reply:
Oh, OK, because these were always the `outputbucket` thanks. I missed that.